Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.7.1-onbuild
+MAINTAINER Jimmy Mesta "jimmy.mesta@gmail.com"
+
+RUN wget https://dl.eff.org/certbot-auto -P /usr/local/sbin && \
```

Comment:
I'm guessing you were downloading certbot so you could get a real cert, but it doesn't appear to be used anywhere in the RUN stanzas.